src/python/national_voter_file/us_states/vt/transformer.py

```diff
@@ -267,6 +263,6 @@ def extract_congressional_dist(self, input_columns):
                 'CONGRESSIONAL_DIST'
         """
 
-        # FIXME: Vermont only has 1 at-large Congressional District. Is this OK as a default value?
-        return {'CONGRESSIONAL_DIST': 'AL'}
-
+        # Vermont only has 1 at-large Congressional District. This is the CensusReporter 
```

Comment:
Just re-triggered the build (not sure why it didn't reflect your changes the first time) and the only thing left is an extra space at the end of this line. Once that's fixed it should pass, and if the GEOID makes sense (I wasn't on that conversation, not sure your thoughts @sahil) it should be good!